### ISSUE TITLE
Farming page plot filename shows the wrong minute component

### ISSUE THREAD

**Issue by okkar** [guydavis/machinaris#185]
**Describe the bug**
Farming page plot filename is not the same as the filename on the filesystem. Farming page repeats the hour component of the filename into the minute component.

**To Reproduce**

1. Get a `<plot_id>` value from Farming page.
2. Run `docker exec -it machinaris chia plots check -g <plot_id>`.
3. Look for `Found plot /<plot_dir>/<plot_filename>` in the output.
4. Compare the plot filename to Farming page.

**Expected behavior**
Plot filename on the filesystem should be displayed.

**System setup:**
 - OS: Ubuntu 20.04
 - Docker version number: 19.03.13
 - Machinaris branch: main
 - Machinaris version number: 0.5.0

**Config**
<details><summary>full configuration</summary>

```
docker run --name=machinaris -d -t -h $(hostname -s) \
-p 8926:8926 -p 8927:8927 -p 8444:8444 -p 8447:8447 -p 6888:6888 -p 6885:6885 \
-e TZ=Asia/Singapore \
-e PUID=1000 \
-e PGID=1000 \
-v /home/okkar/.machinaris:/root/.chia:rw \
-v /mnt/ssd-tmp:/plotting:rw \
-v /home/okkar/plots-dst:/plots-dst:rw \
-v /mnt/plots:'/plots:rw \
-e plots_dir=/plots-dst:/plots/ds1815-chia-plots-volume1:/plots/ds1815-chia-plots-volume2:/plots/ds1815-chia-plots-volume3:/plots/ds410-chia-plots-volume1:/plots/ds410-chia-plots-volume2:/plots/ds410-chia-plots-volume3:/plots/ds410-chia-plots-volume4:/plots/hdd-sda:/plots/hdd-sdb:/plots/hdd-sdc:/plots/hdd-sdd:/plots/hdd-sde:/plots/hdd-sdf \
ghcr.io/guydavis/machinaris
```

</details>

**Additional context & screenshots**
![image](https://user-images.githubusercontent.com/3801075/125890341-7ca5e957-894e-460a-9a97-92c8b00b8613.png)

**Comment by guydavis** [comment]
Wow!  You have an excellent eye for detail.  Thanks for the report.  This will be fixed in the next release.  I will close the ticket when once final release (v0.5.1) is out with this fix.  Thanks again!

**Comment by guydavis** [comment]
Now available in [v0.5.1 release](https://github.com/guydavis/machinaris/releases/tag/0.5.1).
